Kernel/Bus/PCI/Definitions.h

```diff
@@ -14,29 +14,30 @@
 
 namespace Kernel {
 
-#define PCI_VENDOR_ID 0x00            // word
-#define PCI_DEVICE_ID 0x02            // word
-#define PCI_COMMAND 0x04              // word
-#define PCI_STATUS 0x06               // word
-#define PCI_REVISION_ID 0x08          // byte
-#define PCI_PROG_IF 0x09              // byte
-#define PCI_SUBCLASS 0x0a             // byte
-#define PCI_CLASS 0x0b                // byte
-#define PCI_CACHE_LINE_SIZE 0x0c      // byte
-#define PCI_LATENCY_TIMER 0x0d        // byte
-#define PCI_HEADER_TYPE 0x0e          // byte
-#define PCI_BIST 0x0f                 // byte
-#define PCI_BAR0 0x10                 // u32
-#define PCI_BAR1 0x14                 // u32
-#define PCI_BAR2 0x18                 // u32
-#define PCI_BAR3 0x1C                 // u32
-#define PCI_BAR4 0x20                 // u32
-#define PCI_BAR5 0x24                 // u32
-#define PCI_SUBSYSTEM_ID 0x2C         // u16
-#define PCI_SUBSYSTEM_VENDOR_ID 0x2E  // u16
-#define PCI_CAPABILITIES_POINTER 0x34 // u8
-#define PCI_INTERRUPT_LINE 0x3C       // byte
-#define PCI_SECONDARY_BUS 0x19        // byte
+#define PCI_VENDOR_ID 0x00                  // word
+#define PCI_DEVICE_ID 0x02                  // word
+#define PCI_COMMAND 0x04                    // word
+#define PCI_STATUS 0x06                     // word
```

Comment:
to signal the size of somthing, you can 0 padit instead
(idk if unescaped comments in defines are a good idea)

Reply:
I just added a macro to the list of the macros so if we want to fix this, it should be done in a separate commit and PR.
If it's something I've done badly, please tell me what to do. Also, what do you mean by zero padding to signal the size of something?

Reply:
With zero padding I mean, if its a word it should have 4 hex digits, if it is a dword 8

Reply:
ah, I see what you mean, I don't like that approach though... Ideally we should have a fixed C-style array with entries of each valuable register in the PCI config header, and the write/read limits of it so we can enforce it. This will happen in a separate PR... I don't want this PR to be a burden on the reviewers.

Reply:
If you like my idea, it might be a good idea to open an issue about it so I don't forget to implement it in the near future :)